### ISSUE TITLE
Longer text on smaller displays writes over the footer

### ISSUE THREAD

**Issue by Haroenv** [bullgit/logorator#5]
![](https://i.imgur.com/x8toDLn.png) 

probably because of the positioning of the footer is absolute(?)

**Comment by LukyVj** [comment]
Yeah I did nothing for the responsive part so far, feel free to suggest something. I'll add a get strated on the readme. 

If you want to give it a try :)

**Comment by Haroenv** [comment]
I'll see when I have time for this, should be possible
